glide.yaml

```diff
@@ -5,7 +5,7 @@ import:
 - package: github.com/dgrijalva/jwt-go
   version: ^3.0.0
 - package: github.com/goadesign/goa
-  version: ^1.1.0
+  version: be1e8b9a2614909d8805079ebd1c947778170c0e
```

Comment:
@alexeykazakov @hectorj2f Even though it is declared the version as `^1.1.0` the exact version is locked to `be1e8b9a2614909d8805079ebd1c947778170c0e` https://github.com/almighty/almighty-core/blob/master/glide.lock#L42 Now I have put back `^1.1.0` in YAML file and the lock file I kept as such. Is this Okay?